std/testing/asserts_test.ts

```diff
@@ -245,6 +246,35 @@ test("testingAssertFailWithWrongErrorClass", function (): void {
   );
 });
 
+test("testingAssertFailWithReturnType", function (): void {
+  const shouldFail = true;
```

Comment:
what purpose does shouldFail have?

Reply:
![image](https://user-images.githubusercontent.com/21317379/83724750-b8fa1d00-a638-11ea-804f-a9751862acc5.png)

I used it to avoid unreachable code detected during static analysis - but looks like the linter doesn't actually consider this a warning/error - I'll remove! 👍 nice catch 👀